tiledb/sm/c_api/tiledb.h

```diff
@@ -3290,6 +3290,51 @@ TILEDB_EXPORT int32_t tiledb_query_condition_combine(
     tiledb_query_condition_combination_op_t combination_op,
     tiledb_query_condition_t** combined_cond) TILEDB_NOEXCEPT;
 
+/**
+ * Create a query condition representing a negation of
```

Comment:
NIT: this comment could be made to be 2-3 lines.

Reply:
Done.